src/job/MomoJobDescription.ts

```diff
@@ -0,0 +1,19 @@
+import { JobEntity } from '../repository/JobEntity';
+
+export interface SchedulerStatus {
```

Comment:
We could consider extending this info with the number of currently running executions.

Reply:
I think we currently do not have this information (and I don't want to blow up the PR even more ...)
We might have this information when we fix the 'stop job when stopping scheduler' issue, we should keep in mind to extend this then

Reply:
Ah no, of course we have the 'running' property in the database, we can just use that one. I'll check